v2/emailpassword/common-customizations/embed-sign-in-up-form.mdx

```diff
@@ -128,4 +241,46 @@ SuperTokens.init({
 })
 ```
 </TabItem>
-</FrontendSDKTabs>
\ No newline at end of file
+</FrontendSDKTabs>
+
+## Optional: Prevent redirection after SignIn / Up
+
+Upon successful login, SuperTokens redirects the user to the return value of `getRedirectionURL` provided in the recipe config. By default or when returning `undefined` from `getRedirectionURL`, the user is redirected to the `/` route. If you wish to completely prevent redirection, return `null` instead.
```

Comment:
mention here too that this is useful when you want to show the UI in a popup and just dismiss the popup post login.